src/cr/cube/matrix/measure.py

```diff
@@ -1228,6 +1216,14 @@ def blocks(self):
 class _UnweightedCounts(_BaseSecondOrderMeasure):
     """Provides the unweighted-counts measure for a matrix."""
 
+    def __init__(
+        self, dimensions, second_order_measures, cube_measures, diff_nans=False
```

Comment:
Why is there a default value here, for diff_nans, but not in the other class?

Reply:
this is wrong ... my bad